[2024-06-09 21:11:11,036][backoff][INFO] - Backing off chat_completion(...) for 0.2s (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/complete'))
[2024-06-09 21:11:11,329][backoff][INFO] - Backing off chat_completion(...) for 1.7s (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/complete'))
[2024-06-09 21:11:13,132][backoff][INFO] - Backing off chat_completion(...) for 2.4s (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/complete'))
[2024-06-09 21:11:15,614][backoff][INFO] - Backing off chat_completion(...) for 4.3s (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/complete'))
[2024-06-09 21:11:20,052][backoff][INFO] - Backing off chat_completion(...) for 6.4s (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/complete'))
[2024-06-09 21:11:26,572][backoff][INFO] - Backing off chat_completion(...) for 17.9s (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/complete'))
[2024-06-09 21:11:44,628][backoff][INFO] - Backing off chat_completion(...) for 4.7s (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/complete'))
[2024-06-09 21:11:49,426][backoff][ERROR] - Giving up chat_completion(...) after 8 tries (aiohttp.client_exceptions.ClientResponseError: 400, message='Bad Request', url=URL('https://api.anthropic.com/v1/complete'))
